source/_layouts/team-member.blade.php

```diff
@@ -0,0 +1,58 @@
+@extends('_layouts.main')
+
+@section('body')
+    <!-- ====== Banner Start ====== -->
+    <section class="ud-page-banner">
+        <div class="container">
+        <div class="row">
+            <div class="col-lg-12">
+            <div class="ud-banner-content">
+              <h1>{{ $page->name }}</h1>
+              <h3>{{ $page->role }}</h3>
+            </div>
+            </div>
+        </div>
+        </div>
+    </section>
+    <!-- ====== Banner End ====== -->
+
+    
+    <section id="team" class="ud-team">
+        <div class="container">
+          <div class="row">
+            <div class="col-lg-12">
+              <div class="ud-section-title mx-auto text-center">                
+                <p>
+                  {{ $page->bio }} 
+                </p>
+              </div>
+            </div>
+          </div>
+
+          <div class="row">
+            <div class="col-lg-12">
+              <div class="ud-single-team wow fadeInUp" data-wow-delay=".1s">
+                <div class="ud-team-image-wrapper">  
+                  <img
+                    src="{{ $page->baseUrl }}assets/images/team/{{ \Illuminate\Support\Str::slug($page->name) }}.jpg"
+                    alt="{{ $page->name }}"
+                    class="shape shape-1 mb-5"
+                  />
```

Comment:
Suggestion:

Replace the image implementation by Gravatar:

https://github.com/PHPSP/phpsp.org.br/blob/887a733f89283b2e5ae8e802ae1b362bfdbefaeb/source/_partials/content/authors/about-card.blade.php#L5-L10

By this way will be possible remove the photo images from repository.